source/type/data.cpp

```diff
@@ -63,7 +63,7 @@ namespace pt = boost::posix_time;
 namespace navitia {
 namespace type {
 
-const unsigned int Data::data_version = 72;  //< *INCREMENT* every time serialized data are modified
+const unsigned int Data::data_version = 3;  //< *INCREMENT* every time serialized data are modified
```

Comment:
Whoop, it will not be unsafe for a prod deployment ?
I don't know if i'm right, but afet this merge, it will be possible to use a data batch with `data_version > 3` or else it must to have the exact `data_version number` to work ?

Reply:
It's exact version matching: https://github.com/CanalTP/navitia/blob/8c1978085e138de9fd128d40001c1b5e4e4459e1/source/type/data.cpp#L96